bloom-inference-scripts/bloom-ds-inference.py

```diff
@@ -191,6 +191,7 @@ def write_checkponts_json():
     mp_size=world_size,
     base_dir=repo_root,
     dtype=getattr(torch, infer_dtype),
+    quantization_bits=8 if args.dtype == 'int8' else 4,
```

Comment:
what happens with `--dtype float16`? 

probably best to set this in `kwargs` only if quantization dtype is provided

Reply:
The quabtization-bit should not be used when running in half-precision. But, I agree we can do it in the kwargs and only for qunatized inference mode.

Reply:
these demos are already used by many users so let's make those nice and clean configuration-wise, so it's clear to the read when what bits should be enabled.